docs/site/guides/rest/customize-openapi.md

```diff
@@ -0,0 +1,48 @@
+---
+lang: en
+title: 'Customizing how OpenAPI spec is served'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, OpenAPI, Customize
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/rest-customize-openapi-serving.html
```

Comment:
As @raymondfeng stated in the other PRs, we might want to have the file name to begin with capital letter:
```suggestion
permalink: /doc/en/lb4/Rest-customize-openapi-serving.html
```